libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.m

```diff
@@ -535,9 +539,24 @@ - (void)dismissSnapshot
     }
 }
 
+- (BOOL)shouldDisableExternalPaste
```

Comment:
I agree that would be cleaner instead of having if (x || y) logic in random places.